api/src/index.js

```diff
@@ -11,17 +11,88 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
+//Get all meals
 apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+  try {
+    const SHOW_TABLES_QUERY =
+      process.env.DB_CLIENT === "mysql2"
+        ? "SELECT * FROM Meal"
+        : "SHOW TABLES;";
```

Comment:
```suggestion
    const ALL_MEALS_QUERY = SELECT * FROM Meal";
```
No need to keep this database client check.

Reply:
Thanks for your suggestion.